examples/util/openvino/OpenVINOModelImporter.js

```diff
@@ -414,8 +438,17 @@ class OpenVINOModelImporter {
           if (inDims.length === 2) inSize = inDims[1];
           const outputSize = node.getInt('out-size');
           const weightsDims = [outputSize, inSize];
-          const weightsTensor = weights.getInitializer();
-          const biasTensor = bias ? bias.getInitializer() : new Float32Array(outputSize).fill(0);
+          const weightInfo = weights.getTensorInfo();
+          let weightBuffer = raw_buffer.slice(weightInfo.byteOffset, weightInfo.byteOffset + weightInfo.byteLength);
```

Comment:
arraybuffer.slice() would create new memory. Why do we need that?

Reply:
> arraybuffer.slice() would create new memory. Why do we need that?

Because the value of tensor will be incorrect and finally cause wrong predict result without this step.

Reply:
This actually increase the memory consumption.

> Because the value of tensor will be incorrect and finally cause wrong predict result without this step.

Is it a tf.js wasm backend bug?

Reply:
>Is it a tf.js wasm backend bug?

Yes, it's this issue filed upstream [here](https://github.com/tensorflow/tfjs/issues/2998).

Reply:
As it is a tfjs-wasm issue, can we workaround it in TfjsModel wasm path? Workaround here would impact all other backends. What do you think?

Reply:
That makes sense. I'll try it.